types/edtr-io__mathquill/edtr-io__mathquill-tests.ts

```diff
@@ -0,0 +1,39 @@
+import { MQ, Config } from 'edtr-io__mathquill';
```

Comment:
Please use the original name of the package.

```suggestion
import { MQ, Config } from '@edtr-io/mathquill';
```